notebooks/SMAP/1.0 Download SMAP data.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>This could also be achieved using <code>!mkdir &lt;path-to-data&gt;</code>  .  For an interactive notebook, is three calls to <code>os</code> overkill. </p><p> </p><p>We should also show how to change this to some other path. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>